content/en/docs/build/configuration.md

```diff
@@ -3,6 +3,14 @@ title: "Configuration"
 draft: false
 ---
 
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+## Controller Settings
+
 The controller is installed into Kubernetes with reasonable defaults. However, there are some settings that can be overridden using environment variables in [`controller.yaml`](../deploy/500-controller.yaml).
```

Comment:
One of those links that probably was always broken.

Reply:
Yes - this should be changed to be an absolute reference on GitHub.